### ISSUE TITLE
Allow metering and scoping of Server's output when muted

### ISSUE THREAD

**Issue by patrickdupuis** [supercollider/supercollider#3108]
Currently, muting the Server does not allow it's output to be examined though metering or scoping. It would be great if muting the Server killed all sound, while still showing the output level. This would make muting function like it does inside a DAW.

**Comment by LFSaw** [comment]
could be implemented by moving the scope synth before the mute synth. however, we should make the mute state visually noticable from within the scope.

Just as a side note: you can always scope single synths by rerouting their output to a non-HW `AudioBus` and scope that one.

**Comment by brianlheim** [comment]
> we should make the mute state visually noticable from within the scope

why should the scope know about this? wouldn't that require scope to understand when it's being used as a scope on the whole server vs when it's being used on a bus? and mute state is already indicated in other places (server GUI window, IDE). seems like a benefit that isn't worth the entanglement.

**Comment by patrickdupuis** [comment]
I agree with @brianlheim the scope should not have to indicate that the hardware output busses are muted. However, the server meter GUI should have a mute indicator like the server GUI window.
